asgiref/sync.py

```diff
@@ -337,5 +339,33 @@ def get_current_task():
 
 
 # Lowercase is more sensible for most things
-sync_to_async = SyncToAsync
```

Comment:
Hmm, we can't just remove this implementation - I consider SyncToAsync to also be a public interface out of this module. Can you make it function from within there?